contracts/ExtendedSimpleAuthorization.sol

```diff
@@ -0,0 +1,16 @@
+pragma solidity ^0.4.19;
+
+import "./SimpleAuthorization.sol";
+
+// Truffle overloading workaround
+contract ExtendedSimpleAuthorization is SimpleAuthorization {
+
+    function check2(address _token, address _address) public /* view */ returns (uint8 resultCode) {
+        return check(_token,_address);
+    }
+
+    function check4(address _token, address _from, address _to, uint256 _amount) public /* view */ returns (uint8 resultCode) {
```

Comment:
Why `check4` and `check2` with the explicit numbers? Is it idiomatic in Solidity?

Reply:
This "Extended" contract is a workaround for truffle test framework not able to parse overloaded functions.

Reply:
Ooooh, right. I remember this now 😭